src/actors/rqgActor.ts

```diff
@@ -42,7 +43,17 @@ export class RqgActor extends Actor {
     super.prepareEmbeddedDocuments();
     const actorSystem = this.system;
     const { con, siz, pow } = this.actorCharacteristics();
-    actorSystem.attributes.hitPoints.max = RqgCalculations.hitPoints(con, siz, pow);
+    if (con !== null) {
```

Comment:
At some places you check for undefined and other places for null. This can be unified with `value == null` (or `value != null`) since that will work for both `undefined` and `null`. Javascript apparently felt that null was such a great idea that they created two varieties of it :-)

Reply:
Wow " !== null " does not pick up undefined but " != null" does.  Crazy.

Reply:
If I want to set the attributes.hitPoints to "nothing" I should still do "attributes.hitPoints = undefined" though, right?

Reply:
Done